sellotape/sellotape_dj/main_app/templates/sellotape/user.html

```diff
@@ -0,0 +1,64 @@
+{% extends 'sellotape/base.html' %}
+
+{% load static %}
+
+{% block customlinks %}
+<link rel="stylesheet" type="text/css" href="{% static 'sellotape/user.css' %}">
+{% endblock %}
+
+{% block content %}
+    <div class="header">
+        <a href="{% url 'main_app:user' username=profile.username %}">
+            <h2>{{ profile.first_name }} {{ profile.last_name }}</h2>
+        </a>
+        {% if live_stream %}
+            <span class="red-dot"></span>
+        {% endif %}
+    </div>
+
+    {% if live_stream %}
+        <div class="section">
+            {{ profile.first_name}} is live now at
+            <a class="live-link" href="{{ live_stream.link }}">{{ live_stream.link }}</a>.
+        </div>
+    {% endif %}
+
+    {% if future_streams|length %}
+        <div class="section">
+            <h3>Future Streams</h3>
+            <ul class="streams">
+                {% for stream in future_streams %}
+                    <li>
+                        <h4>{{ stream.title }}</h4>
+                        <p>
+                            Airs on {{ stream.airs_on|date }} <a href="{{ stream.link }}">here</a>.
```

Comment:
I think we should embed the streams on our website,
the way to do it is by identifying the platform (e.g. YouTube and Twitch) or by adding a 'platform' field to 'Streams DB',
and then using their embedding format.

Reply:
That could be cool, but in my opinion we should consider it later as an enhancement feature and first have a simpler platform. The Stream Model can always be modified :)